README.md

```diff
@@ -14,6 +14,15 @@ is provided. To simplify the process of obtaining a new access token, there is a
 utility tool you can use. It is recommended that the token is provided via the
 `MEGAPORT_TOKEN` environment variable.
 
+Firstly, export the follwing vars:
+```sh 
+export MEGAPORT_USERNAME=your-user-name
+export MEGAPORT_PASSWORD=your-password
+And: 
+export MEGAPORT_ENDPOINT=api.EndpointStaging #For Dev (Staging) 
+Or:
+export MEGAPORT_ENDPOINT=api.EndpointProduction # For Production
```

Comment:
This would not work, you have to export the actual value, eg: `https://api.megaport.com`.